src/components/Todo.js

```diff
@@ -0,0 +1,43 @@
+import { useState } from 'react';
+import Form from './Form';
+import Item from './Item';
+
+export default function Todo() {
+  const [savedList, setSavedList] = useState([]);
+  let [toDoList, setToDoList] = useState([]);
+  let [doneList, setDoneList] = useState([]);
+  toDoList = savedList.filter((todo) => todo.isDone === false);
+  doneList = savedList.filter((todo) => todo.isDone === true);
```

Comment:
`useState`는 `const`로 선언하며, 이렇게 변경된 값을 직접 할당하는 것보다 `setState`를 이용하셔야 합니다. 또한 `useEffect`를 사용하지 않을 경우 too many re-renders 에러가 발생하기 때문에 아래처럼 사용하셔야 합니다. 

```suggestion
 const [toDoList, setToDoList] = useState([]);
  const [doneList, setDoneList] = useState([]);

  useEffect(() => {
    setToDoList(savedList.filter((todo) => todo.isDone === false));
    setDoneList(savedList.filter((todo) => todo.isDone === true));
  }, [savedList]);
```
`useState`와 `useEffect`에 대해 더 알아보시면 좋을 것 같아요~

Reply:
네 알려주셔서 감사합니다!! 🙌